modules/common/css/lib/accessibility.scss

```diff
@@ -21,3 +21,16 @@
     outline: none;
   }
 }
+
+.wdc-accessible-hide {
```

Comment:
Can you turn this into a mixin and call it from within `.wdc-accessible-hide` so it's available as both a mixin and a class?

Reply:
updated!